proxy/src/AuthClient.ts

```diff
@@ -38,18 +42,26 @@ export class AuthClient {
     return this._client;
   }
 
-  private async generateAccessToken(): Promise<AccessToken> {
+  private async _generateAccessToken(): Promise<AccessToken> {
     const scope = this._scopes;
-    if (scope.includes("openid") || scope.includes("email") || scope.includes("profile") || scope.includes("organization"))
-      throw new Error("Scopes for an Agent cannot include 'openid email profile organization'");
+    if (
+      scope.includes("openid") ||
```

Comment:
silly prettier rules ;)

Reply:
yea i know... a lot of the changes are from prettier, i can revert if you want, just lmk

Reply:
Nah that's fine you can leave them, just thought i'd complain a bit :)